openpower/package/occ/0001-Use-std-gnu89-the-gcc-4.8-default.patch

```diff
@@ -0,0 +1,60 @@
+From c103611841ea1d28bb8850b415bae2997af93981 Mon Sep 17 00:00:00 2001
+From: Stewart Smith <stewart@linux.vnet.ibm.com>
+Date: Thu, 3 Nov 2016 15:34:38 +1100
+Subject: [PATCH] Use std=gnu89, the gcc 4.8 default
+
+Signed-off-by: Stewart Smith <stewart@linux.vnet.ibm.com>
```

Comment:
Done and done. Well.. I copied the commit-msg hook over and it seemed to add some ID, so I'm hoping that's correct :)